dips/dip-158.md

```diff
@@ -0,0 +1,795 @@
+---
+dip: 158
+title: P2M (eCommerce) Payments Off-chain API
+authors: Daniel Prinz, Yinon Zohar, Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 03/15/2021
+---
+
+# Summary
+
+This DIP is an extension of the Off-Chain protocol for enabling eCommerce P2M payments (Peer To Merchant) of the following types:
+* Charge
+* Auth / Capture
+
+This DIP defines the type of eCommerce P2M payments, corresponding requests and the payment structures. 
+ 
+This DIP extends [DIP-1](https://dip.diem.com/dip-1/) which specifies P2P payment flows (Peer To Peer).
```

Comment:
This is a bit unclear, how does it extend it?

Reply:
Objects from DIP-1 that are used in this DIP are extended with new properties (e.g. additional statuses)